JSDemos/Demos/DateBox/Overview/description.md

```diff
@@ -4,7 +4,16 @@ The [DateBox](/Documentation/ApiReference/UI_Components/dxDateBox/) editor helps
 A value the DateBox displays.
 
 - [type](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#type)       
-Specifies whether the DateBox allows users to select only date, only time, or both.
+Specifies whether the DateBox allows users to select one of the following types:
+
+  - *"date"*    
+  Users can select the date from the calendar, or they can type in their own values.
+
+  - *"time"*    
+  Users can select from the time slots that lay in range from 12 AM to 11:30 PM with a step of 30 minutes, or they can type in their own values. Specify the [interval](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#interval) property to set the different time step.
+
+  - *"datetime"*    
+  Users can select both the date from the calendar and the time. The **Today** button is available in the date picker.
 
 - [displayFormat](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#displayFormat)        
 A date/time display format. You can use one of the [predefined formats](/Documentation/ApiReference/Common/Object_Structures/format/#type) or specify a [custom format](/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String). This demo illustrates the latter.
```

Comment:
This line is fine, but I'm not able to add comments to the lines below. Can you please make the following change?

Line 13:
Specifies the type of UI used to select date or time. This demo shows how to change calendar to roller date picker.

Can we update this line as follows?

`Specifies the type of UI used to select a date or time. This demo shows how to change a calendar to a roller date picker.`

Also, is "roller date picker" the correct technical term? Or would it be better to say "rolling" or "scrolling" date picker? Just checking.

Reply:
Yes, it's a correct term.

Reply:
Thank you.